src/pages/Privacy.vue

```diff
@@ -0,0 +1,130 @@
+
+                  <h3>Disclosure and Transfer of Your Personal Data</h3>
+
+                  <p>We do not sell, rent or otherwise disclose personally identifiable information with any other parties other than as set out in this Privacy Policy. Your data is stored securely in line with data protection law. The Data Protection Act 1998 does however permit certain disclosures without consent when the information is requested for one or more of the following purposes:</p>
+
+                  <ul>
+                      <li>To safeguard national security;</li>
+                      <li>Prevention or detection of crime including the apprehension or prosecution of offenders;</li>
+                      <li>Assessment or collection of tax duty;</li>
+                      <li>Discharge of regulatory functions (includes health, safety and welfare of persons at work);</li>
+                      <li>To prevent serious harm to a third party;</li>
+                      <li>To protect the vital interests of the individual, this refers to life and death situations.</li>
+                  </ul>
+
+                  <p>All requests to provide personal data for one of the above reasons will be specifically authorised by ARC’s Business Administrator.</p>
+
+                  <h3>Use of Cookies and Other Technologies</h3>
+
+                  <p>A Cookie is a small piece of data containing a unique reference which is written to a web-enabled user device such as a computer, smartphone, tablet and/or other device running a web browser capable of connecting you to the internet. ARC uses this technology to collect information about Rosie users in order to provide them with a better experience by recognising each user and informing him/her of any new messages since their previous visit. ARC also uses Google Analytics to gather non-personally identifiable information on our visitors. This information helps us understand how Rosie is being used and is gathered through cookies and code embedded in Rosie.</p>
+
+                  <h3>Cookie Consent</h3>
+
+                  <p>To comply with the law, we need to ask for your consent to set the Cookies described above. The first time you use Rosie, a pop-up message will appear asking for your consent to place cookies on your device.  In order to provide your consent, please click ‘continue’.  Once your consent has been provided, this message will not appear again, unless we make any changes to our practices and need to ask for your consent.  If you want to withdraw your consent, you can do so by altering the browser settings on your device. For more information about cookies, go to <a href="http://www.allaboutcookies.org" class="info" target="_blank">www.allaboutcookies.org</a> and <a href="www.youronlinechoices.com/uk" class="info" target="_blank">www.youronlinechoices.com/uk</a></p>
```

Comment:
Hmm - not sure if I feel comfortable amending Russ's documentation so I will raise these comments with him on the Trello card.